### PR TITLE
#457 3rd party cookie alert txt change

### LINKED ISSUE #457: Third party cookies error message
Change this error message to include unchecking "prevent cross-site tracking" in Safari
unnamed (4)

### PR DESCRIPTION
**Pull request by pushyamig**
Fixes #457 #441 

Test Plan 
#457 : To reproduce the error launch CCM from Safari
#441 : Open Add Non UM user in the Bulk option and see the link is added on the upload CSV page

### CODE REVIEW

**Comment by pushyamig** [comment]
I have added the #441 to the PR as well since the change is about the txt update

File: ccm_web/client/src/pages/LaunchError.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ export default function LaunchError (props: LaunchErrorProps): JSX.Element {
   const iOSLink = 'https://www.whatismybrowser.com/guides/how-to-enable-cookies/safari-iphone'
   const firefoxLink = 'https://support.mozilla.org/en-US/kb/enhanced-tracking-protection-firefox-desktop?redirectslug=enable-and-disable-cookies-website-preferences'
   const edgeLink = 'https://www.whatismybrowser.com/guides/how-to-enable-cookies/edge'
+  const safariLink = 'https://support.apple.com/guide/safari/prevent-cross-site-tracking-sfri40732/mac'
```
Comment (jaydonkrooss) [inline ccm_web/client/src/pages/LaunchError.tsx:17]: nitpick but trying to distinguish 3 safari-related links might be unintuitive. maybe something like `safariMacOSLink`, `safariIOSLink`, `safariCrossSiteTrackingLink`?

**Comment by jaydonkrooss** [comment]
other than my small variable suggestion, changes looks good

File: ccm_web/client/src/pages/LaunchError.tsx (hunk 1 above)
Comment (pushyamig) [reply ccm_web/client/src/pages/LaunchError.tsx]: Done

**Review by jaydonkrooss — APPROVED** [review]

**Pull request merged**